doc/install.rst

```diff
@@ -147,13 +152,11 @@ Alternatively, you can install the latest development version from
 
     pip install --pre --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pygmt
 
-or from PyGMT's `GitHub repository <https://github.com/GenericMappingTools/pygmt>`__ 
-(slower as it downloads the whole archive)::
-
-    pip install git+https://github.com/GenericMappingTools/pygmt.git#egg=pygmt
```

Comment:
Just double checking that we really want to remove this install option?

Reply:
For most users, it's the same as 
```
pip install --pre --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pygmt
```
but slower. 

I don't see any reasons to keep it.

Reply:
Ok, fine with removing it. I just think the `pip install git+https` option is nice when you want to install from a GitHub branch (but that's not what we're doing here).